src/web/client/schema/constants.ts

```diff
@@ -30,6 +30,8 @@ export enum schemaEntityKey {
     MAPPING_ENTITY_FETCH_QUERY = "_mappingEntityFetchQuery",
     EXPORT_TYPE = "_exporttype",
     ATTRIBUTES = "_attributes",
+    CONTENT = "_content",
+    ROOT_WEB_PAGE_ID = "_rootwebpageid",
```

Comment:
this is the only place you need this constant and not below. Add _adx_rootwebpageid_value for V1 and content.rootwebpageid here.